### PR TITLE
{compute-diagnostic} remove compute-diagnostic command group

### PR DESCRIPTION
**Pull request by joal42**
compute-diagnostic command group is used for az cli extension.
Now, it replaced with compute-recommenderr command group in az cli. Remove this command group to avoid confusion

### CODE REVIEW

File: Commands/tree.json [hunk 1]
```diff
@@ -18290,65 +18290,6 @@
...
         ]
       },
-      "compute": {
-        "help": {
-          "short": "Mange azure compute vm config"
-        },
-        "names": [
-          "compute"
-        ]
-      },
```
Comment (kairu-ms) [inline Commands/tree.json:18300]: Why remove the compute command group?

File: Commands/compute-diagnostic/spot-placement-recommender/_generate.md [hunk 2]
```diff
@@ -1,16 +0,0 @@
-# [Command] _compute-diagnostic spot-placement-recommender generate_
-
-Generates placement scores for Spot VM skus.
-
-## Versions
-
-### [2021-06-01-preview](/Resources/mgmt-plane/L3N1YnNjcmlwdGlvbnMve30vcHJvdmlkZXJzL21pY3Jvc29mdC5jb21wdXRlL2xvY2F0aW9ucy97fS9kaWFnbm9zdGljcy9zcG90cGxhY2VtZW50cmVjb21tZW5kZXIvZ2VuZXJhdGU=/2021-06-01-preview.xml) **Stable**
```
Comment (kairu-ms) [inline Commands/compute-diagnostic/spot-placement-recommender/_generate.md:7]: Please also remove the following two files as well [2021-06-01-preview.json](https://github.com/Azure/aaz/tree/main/Resources/mgmt-plane/L3N1YnNjcmlwdGlvbnMve30vcHJvdmlkZXJzL21pY3Jvc29mdC5jb21wdXRlL2xvY2F0aW9ucy97fS9kaWFnbm9zdGljcy9zcG90cGxhY2VtZW50cmVjb21tZW5kZXIvZ2VuZXJhdGU=/2021-06-01-preview.xml) and [2021-06-01-preview.xml](https://github.com/Azure/aaz/tree/main/Resources/mgmt-plane/L3N1YnNjcmlwdGlvbnMve30vcHJvdmlkZXJzL21pY3Jvc29mdC5jb21wdXRlL2xvY2F0aW9ucy97fS9kaWFnbm9zdGljcy9zcG90cGxhY2VtZW50cmVjb21tZW5kZXIvZ2VuZXJhdGU=/2021-06-01-preview.json)

Comment (joal42) [reply Commands/compute-diagnostic/spot-placement-recommender/_generate.md:7]: thanks. removed

File: Commands/tree.json (hunk 1 above)
Comment (joal42) [reply Commands/tree.json]: revert removing. keep it.

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**